### ISSUE TITLE
"Nothing to annotate" in rails 4.2

### ISSUE THREAD

**Issue by ekampp** [ctran/annotate_models#173]
Disclaimer: I'm aware that 4.2 is in alpha at the moment.

After upgrading to rails 4.2.0.alpha, annotation always yields `Nothing annotated.`

Running it with the `bundle exec annotate --trace` parameter yields the following:

``` bash
State :complete for :levelup_state is already defined in :state
/Users/emil/.rbenv/versions/2.1.0/lib/ruby/gems/2.1.0/gems/closure_tree-4.4.0/lib/closure_tree/support.rb:29: warning: already initialized constant FeatHierarchy
/Users/emil/.rbenv/versions/2.1.0/lib/ruby/gems/2.1.0/gems/closure_tree-4.4.0/lib/closure_tree/support.rb:29: warning: previous definition of FeatHierarchy was here
Nothing annotated.
```

As you can see, the warnings are unrelated to Annotate, and yields nothing of interest indicating why the annotation didn't find any models to annotate, even though there's several models in the project.

**Comment by dlynam** [comment]
I'm seeing this issue in Rails 4.2.0 (Ruby 2.2.0) still. I've tried multiple versions (v2.6.6 and v2.6.7). Could this still be an issue?

**Comment by ekampp** [comment]
@dlynam are you sure it's the exact same issue? I haven't used this gem for a while, so I don't know if this is still an issue. Can you track down, where your exact problem arises from?

**Comment by dlynam** [comment]
@ekampp looks like this issue has already been fixed, just not pushed to -v2.6.7 https://github.com/ctran/annotate_models/issues/235

**Comment by bcackerman** [comment]
Running `gem 'annotate', '2.6.5'` and it works. I rolled back from `2.7.1`
